pydatastructs/miscellaneous_data_structures/tests/test_queue.py

```diff
@@ -35,4 +33,7 @@ def test_Queue():
     assert q1.popleft().data == 2
     assert q1.popleft().data == 3
     assert len(q1) == 0
-    raises(ValueError, lambda: q1.popleft())
+    raises(IndexError, lambda: q1.popleft())
+
+    q1 = Queue(implementation='linkedlist',items=['a',None,type,{}])
```

Comment:
Can we have `linked_list` as the implementation?

Reply:
Just minor renaming at some places is needed.

Reply:
Sure